src/containers/Pages/AboutPage/AboutPage.module.scss

```diff
@@ -0,0 +1,126 @@
+@import "../../../assets/designs/commonTheme.scss";
+
+.content {
+  background-color: $backgroundColor;
+  height: fit-content;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+}
+
+p {
+  text-align: justify;
+  text-justify: inter-word;
+  color: #5f5f5f;
```

Comment:
if its a shared colour try to refer to the existing colour theme file

Reply:
will do